python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/vxlan_interface.schema.yml

```
@@ -119,6 +119,9 @@ keys:
                       items:
                         type: str
                         description: Remote VTEP IP Address.
+                    flood_group_multicast_ipaddress:
```

This is not similar to the CLI.
```suggestion
                    flood_group:
```

---

Fixed!